packages/cli/src/cli.ts

```diff
@@ -52,6 +53,14 @@ program
         await promptNewIntegration(path.resolve(process.cwd(), dirPath));
     });
 
+program
+    .command('dev')
+    .description('run the integrations dev server')
+    .option('-p, --port <number>', 'port to run the dev server on')
```

Comment:
How can the user chooses the space to dev with?

Reply:
That's where `gitbook-dev.yaml` comes into picture